java/code/src/com/redhat/rhn/domain/notification/test/NotificationFactoryTest.java

```diff
@@ -32,7 +33,7 @@
     public final void testVisibilityForNoRoles() {
         assertEquals(0, UserNotificationFactory.unreadUserNotificationsSize(user));
         NotificationMessage msg = UserNotificationFactory.createNotificationMessage(new OnboardingFailed("minion1"));
-        UserNotificationFactory.storeNotificationMessageFor(msg, Collections.emptySet());
+        UserNotificationFactory.storeNotificationMessageFor(msg, Collections.emptySet(), Optional.empty());
```

Comment:
Personal preference: I always statically import `empty`, `of`, `emptySet` for legibility - ymmv